engine_modules/corporation/tasks.py

```diff
@@ -10,11 +12,28 @@ class CrashCorporationTask(ResolutionTask):
 	RESOLUTION_ORDER = 850
 
 	def run(self, game):
-		corporations_to_crash = game.corporation_set.filter(assets__lte=0)
+		corporations_to_crash = list(game.corporation_set.filter(assets__lte=0))
 
 		if not corporations_to_crash:
 			return
 
+		first_crash_taurus = False
+		# Handle the case of Taurus Part 1
+		for corporation in corporations_to_crash:
+			if corporation.base_corporation.slug == 'taurus':
```

Comment:
we can't do that this way it's too specifically tied to the current corporation.

I'd rather add a "phoenix" flag on the corporation definition, so we can use this behavior on other coprorations too